layouts/footer.slim

```diff
@@ -1,6 +1,35 @@
-footer.ui.center.aligned.container lang='en'
-  hr
-  p Copyright © 2016 by PPType, Inc.
+.ui.footer.black.inverted.segment lang='en'
+  .ui.left.aligned.container
+    .ui.stackable.inverted.grid
+      .three.wide.column
+        h4.ui.inverted.header Powered By
+        .ui.inverted.link.list
+          a.item href="http://semantic-ui.com" target="_blank" Semanti-UI
+          a.item href="https://bootswatch.com" target="_blank" Bootswatch
+          a.item href="https://nanoc-plate.github.io" target="_blank" Nanoc-plate
+
+      .three.wide.column
+        h4.ui.inverted.header Documentation 
+        .ui.inverted.link.list
+          a.item href="http://semantic-ui-forest.com/help" target="_blank" Get Started 
+          a.item href="http://semantic-ui-forest.com/faq" target="_blank" FAQ
+
+      .three.wide.column
+        h4.ui.inverted.header About
+        .ui.inverted.link.list
+          a.item href="http://semantic-ui-forest.com/contact" target="_blank" Contact 
+          a.item href="http://semantic-ui-forest.com/author" target="_blank" Author
+          a.item href="http://semantic-ui-forest.com/license" target="_blank" License 
+
+      .seven.wide.right.floated.column
+        .ui.compact.left.floated.inverted.segment
+          a.item href="#"
+            img.ui.tiny.left.floated.image src="/static/logo/gravatar/gravatar.svg"
+        .ui.left.floated.inverted.segment
+          a.item href="#"
```

Comment:
Prefer `a.item href="/"` over `a.item href="#"`, since `href="#"` will return to the top of the current page, while `href="/"` will return to the root index page of the website.